src/components/textfieldControl/css/TextfieldStepper.css

```diff
@@ -23,13 +23,20 @@
 /* Textfield Styling */
 
 .fl-textfieldStepper .fl-textfieldStepper-textField {
+    -moz-appearance: textfield;
     font-family: inherit;
     font-size: 100%;
     margin: 0 0.5rem;
     text-align: center;
     width: 7rem;
 }
 
+.fl-textfieldStepper .fl-textfieldStepper-textField::-webkit-inner-spin-button,
+.fl-textfieldStepper .fl-textfieldStepper-textField::-webkit-outer-spin-button {
+    appearance: none;
```

Comment:
It appears this needs to be `-webkit-appearance: none;` for it to work on Safari.

How it looks as it currently is:
<img width="318" alt="Number input in Safari" src="https://user-images.githubusercontent.com/556024/118131551-ab165580-b3cc-11eb-8224-1993eaae0a8c.png">

Reply:
Resolved in 33aa7dc.